packages/kit/src/runtime/app/forms.js

```diff
@@ -65,6 +65,17 @@ export function enhance(form, submit = () => {}) {
 
 		const data = new FormData(form);
 
+		if (DEV && form.enctype !== 'multipart/form-data') {
```

Comment:
```suggestion
		if (DEV && /** @type {HTMLFormElement} */ (HTMLElement.prototype.cloneNode.call(form)).enctype !== 'multipart/form-data') {
```

There are some DOM clobbering problems with direct property access on forms, but it could just be fine as is because it's dev only.

Reply:
I think it's still worthwhile to guard against this, even in dev mode. False positives would still be annoying.

Reply:
Ahhh, nevermind -- some googling got me the answer I wanted

Reply:
If you have
```html
<form><input name=enctype></form>
```

form.enctype then gives you a reference to the input element instead of the enctype of the form